### DISCUSSION TITLE
Case insensitive equals

### DISCUSSION THREAD

**Discussion by VladimirGK** [MorphiaOrg/morphia#2443] [Q&A] [answered]
This was possible in Morphia 1.6.1:
`.field("key").equalIgnoreCase("value")`

I don't see a way of doing that in Morphia 2.2.10, am I missing something?

**Comment by evanchooly** [comment, marked as answer]
You are not missing anything.  What was happening with 1.6.1, is that Morphia would create a case-insensitive `regex` filter for you.  This has hidden performance impacts and was dropped from 2.x for this and other reasons.  To achieve the same thing in 2.0, you can use a regex filter directly or craft your indexes appropriately.  For a broader discussion on the pros and cons of these options, check out [this video](https://www.mongodb.com/developer/products/mongodb/schema-design-anti-pattern-case-insensitive-query-index/) which goes in to a fair bit of detail.
